envs/example/ci-full-rhel/group_vars/all.yml

```diff
@@ -17,6 +17,13 @@ etc_hosts:
 
 neutron:
   enable_external_interface: True
+  l3ha:
```

Comment:
I have a feeling that we will need to set `arp_responder: False` as well here for ping to work.